gulpfile.js

```diff
@@ -88,7 +88,11 @@ gulp.task('build:styles', () => {
 gulp.task('build:standalone', () => {
   const prettyJs = gulp.src('dist/sweetalert2.js')
   const prettyCssAsJs = gulp.src('dist/sweetalert2.css')
-    .pipe($.css2js())
+    .pipe($.css2js({
```

Comment:
These options will "uglify" CSS and make it unreadable for humans. If we're doing so, would it be better to just use minified css? See my suggestion above ^

Reply:
> However, when people are using the .all solution, the change that they are going to debug the CSS in JS is very slim.
> 
> Taking that in account I would prefer using the .min file indeed.

Agree, please go ahead with that.

Reply:
I already did :) By accepting your change!
For my account this PR is ready for merge.